README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/rootstrap/active-storage-base64.svg?branch=master)](https://travis-ci.org/rootstrap/active-storage-base64)
```

Comment:
I think we should also delete `.travis.yml` and any other references to Travis (if any)